src/main/proto/org/hyades/repometaanalysis/v1/repo_meta_analysis.proto

```diff
@@ -25,6 +25,17 @@ message AnalysisResult {
 
   // When the latest version was published.
   optional google.protobuf.Timestamp published = 4;
+  optional IntegrityMeta integrity_meta = 5;
+}
+
+message IntegrityMeta {
+  optional string md5 = 1;
+  optional string sha1 = 2;
+  optional string sha256 = 3;
+  optional string sha512 = 4;
+  string integrity_meta_source_url = 5;
+  // When the component current version last modified.
+  optional google.protobuf.Timestamp current_version_published = 6;
```

Comment:
Need to update these to the latest fields
`message IntegrityMeta {
  optional string md5 = 1;
  optional string sha1 = 2;
  optional string sha256 = 3;
  optional string sha512 = 4;
  // When the component current version last modified.
  optional google.protobuf.Timestamp current_version_last_modified = 5;
  // Complete URL to fetch integrity metadata of the component.
  optional string repository_url = 6;
}`

Reply:
done :)